python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py

```diff
@@ -189,6 +191,14 @@ def transform_vp_timestamps(
         "vp_stop_timestamp"
     ].astype("Int64")
 
+    # change vehicle_consist to list of dicts to list of strings
+    vehicle_positions["vehicle_consist"] = vehicle_positions[
+        "vehicle_consist"
+    ].map(
+        lambda list_of_dicts: [str(item["label"]) for item in list_of_dicts],
+        na_action="ignore",
+    )
+
```

Comment:
`vehicle_consist` is stored as list of dict objects in parquet, this converts to list of strings for loading into RDS.